### ISSUE TITLE
Bug in jump search space

### ISSUE THREAD

**Issue by kudkudak** [msmbuilder/osprey#129]
I think jump search space should differently handle right endpoint?

```{python}
s = SearchSpace()
s.add_jump(min=0, max=1, step=0.1, name="x")
print(s)
```

returns

```{python}
Hyperparameter search space:
  x                        	(enum)    choices = (0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0, 1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.7, 1.8, 1.9)
```

**Comment by cxhernandez** [comment]
Good catch!

**Comment by msultan** [comment]
fixed in #130 i think
